sql/core/src/test/resources/sql-tests/inputs/pgSQL/window_part1.sql

```diff
@@ -0,0 +1,343 @@
+-- Portions Copyright (c) 1996-2019, PostgreSQL Global Development Group
+--
+-- Window Functions Testing
+-- https://github.com/postgres/postgres/blob/REL_12_BETA3/src/test/regress/sql/window.sql
```

Comment:
Can you add line numbers?: https://github.com/apache/spark/blob/a6a663c4379390217443bc5b6f75873fb1c38c73/sql/core/src/test/resources/sql-tests/inputs/pgSQL/aggregates_part1.sql#L6

Reply:
Unfortunately,`REL_12_BETA4` is released.
> https://github.com/postgres/postgres/blob/REL_12_BETA4/src/test/regress/sql/window.sql

Reply:
I can update to match Beta4, but:
- If the PgSQL version is *that* relevant, then we will add a new JIRA for every new PgSQL release for every .sql migration (i.e:  strings.sql, union.sql, date.sql, aggregates.sql, etc). But we didn't.
- If the PgSQL is not that relevant, then there's no need to change PRs to match the latest release candidate version (although it matches a stable version).

What you think?

Reply:
Have you checked the last discussion? https://github.com/apache/spark/pull/24850#discussion_r300215383
I think we don't need to stick to the single snapshot, so its ok to update it to `REL_12_BETA4` in this port. Actually, no difference between `BETA2` and `BETA4` in `window.sql`?

Reply:
No, I didn't get noticed that you guys were already discussing about updating other tests.
I checked last week and there are some major differences between BETA2 and BETA4 for `window.sql`, I just can't remember if lines L1-L319 are impacted by them. So, it's ok to keep at REL2 or should I updated to REL3?